deel/lip/layers.py

```diff
@@ -967,10 +967,19 @@ def build(self, input_shape):
     def _compute_lip_coef(self, input_shape=None):
         return np.sqrt(np.prod(np.asarray(self.pool_size)))
 
+    @staticmethod
+    @tf.custom_gradient
+    def _sqrt(x):
+
+        def grad(dy):
+            return dy / (2*tf.sqrt(x+0.001))
```

Comment:
It's kind of a big ε no?

Reply:
yes, 1/(2*sqrt(0.001)) = 15.
This could be set to lower values : 1/(2*10e-6) = 500 but i dont know if it is enough to avoid numerical instabilities. I set it to 10e-3 i order to be sure.